relay.go

```diff
@@ -379,6 +382,9 @@ func (r *Relayer) handleCallReq(f lazyCallReq) error {
 	origID := f.Header.ID
 	destinationID := remoteConn.NextMessageID()
 	ttl := f.TTL()
+	if ttl > r.maxTimeout {
+		ttl = r.maxTimeout
```

Comment:
Resolved offline - HB *does* rewrite the TTL in the in-flight frame. We'll do the same here.